src/lib/crypto/kimchi_bindings/stubs/binding_generation/src/main.rs

```diff
@@ -35,143 +36,176 @@ use wires_15_stubs::{
 fn main() {
```

Comment:
can you update the `README.md` in the stubs to document the new usage for this CLI?